packages/packages/recently-edited/src/hooks/use-recent-posts.ts

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from 'react';
+import { DocType } from '../types';
+import apiFetch from '@wordpress/api-fetch';
+import { addQueryArgs } from '@wordpress/url';
+import { Document } from '@elementor/documents';
+
+export interface Post {
+	id: number,
+	title: string,
+	edit_url: string,
+	type: {
+		post_type: string,
+		doc_type: DocType,
+		label: string,
+	},
+	date_modified: number,
+}
+
+export default function useRecentPosts( document: unknown ) {
```

Comment:
If you need only document id, just use documentId, and then you can type it as number